news/6295.bugfix

```diff
@@ -0,0 +1 @@
+Redacts the URL of the extra index when `pip -v` to hide the user's password
```

Comment:
Since the URL isn't being removed but only the password, it would be better to say something like, "Redact the password from the extra index URL when using ``pip -v``." Also, you want to use double backticks before and after "pip -v" rather than single, and end the sentence with a period.

Reply:
you mean like \`\`pip -v\`\`?

Reply:
Yeah. I couldn't get it to render in GitHub's viewer when I first tried. This is because the contents are reStructuredText: https://pip.pypa.io/en/stable/development/contributing/#contents-of-a-news-entry

Reply:
I used a backslash before the backtick to escape it and let Markdown ignore it